config/locales/en.yml

```diff
@@ -1671,13 +1671,13 @@ en:
       workflow_version: Workflow Version
     table:
       empty:
-        description: There are no workflow executions associated with this submitter.
-        title: No workflow executions.
+        description: There are no pipelines associated with this submitter.
```

Comment:
**nitpick (blocking):** Can you revert the following 2 translation changes.

Reply:
Updated in 51f582d